recipes/boost/all/conanfile.py

```diff
@@ -258,7 +262,7 @@ def _python_includes(self):
                 if os.path.isfile(python_h):
                     self.output.info('found Python.h: %s' % python_h)
                     return candidate.replace('\\', '/')
-        raise Exception("couldn't locate Python.h - make sure you have installed python development files")
+        raise ConanInvalidConfiguration("couldn't locate Python.h - make sure you have installed python development files")
```

Comment:
Why should this be an invalid configuration? You normally raise this kind of error when something it is not supported, but ins this case it should fail because it does not found the python header

Reply:
changed back (by @Croydon )